pr_agent/tools/pr_description.py

```diff
@@ -74,7 +74,7 @@ async def run(self):
 
             await retry_with_fallback_models(self._prepare_prediction)
 
-            get_logger().info(f"Preparing answer {self.pr_id}")
+            get_logger().info(f"Preparing description regarding {self.pr_id}")
```

Comment:
**Suggestion:** Use a more specific log level, such as `debug`, for the message that indicates a process is starting, to differentiate from completion messages.
```suggestion
            get_logger().debug("Preparing description regarding", extra={"pr_id": self.pr_id})
```